MarsMission/Views/WeatherView.swift

```diff
@@ -8,7 +8,6 @@
 import Foundation
 
 protocol WeatherView {
-	func populateWeatherData(_ lastUpdated: String,_ weatherStation: String)
+	func populateWeatherData(_ lastUpdated: String,_ weatherStation: String,_ date: String,_ temp: Double)
```

Comment:
Why do you have this function if you're using a collection view? Please ensure that you understand how the UI Components that you are using works before you use them